specs/joins.toml

```diff
@@ -8,6 +8,9 @@ statements = [
     "optimize table articles with (max_num_segments = 1)",
 ]
 
+[session_settings]
+optimizer_equi_join_to_lookup_join = "true"
```

Comment:
do we need it `true` here? This can make joins which were executed with hashjoin all this time to be executed with lookup join now, no? Isn't it enough to set it only to the other `lookup_join.toml` file?

Reply:
Why would we disable a join optimization in a join benchmark ? For the query
"select * from articles inner join colors on articles.id = colors.id where colors.id = -1"
this optimization is significant.
<img width="602" alt="Screenshot 2024-08-28 at 11 24 18" src="https://github.com/user-attachments/assets/46220584-ece1-4a9b-9469-4a75c6293988">